src/coreclr/inc/gcinfodecoder.h

```diff
@@ -275,6 +276,7 @@ class BitStreamReader
         m_InitialRelPos = other.m_InitialRelPos;
         m_pCurrent = other.m_pCurrent;
         m_RelPos = other.m_RelPos;
+        m_current = other.m_current;
```

Comment:
On 64bit one native word can act as a "buffer" for quite a few reads when each read take only a few bits. This change reduces the need for indirect reads from the bitstream and may allow the compiler to enregister the "buffer".